src/DocumentElements/documentelements.table.proto

```diff
@@ -114,6 +114,7 @@ message ProtoTableCell {
   ProtoTableCellSettings settings = 3;
   ProtoBoxedUint32 cellConfigNumber = 4;
   ProtoBoxedUint32 index = 5;
+  ProtoTableCell mergedHorizontal = 6;
```

Comment:
Is this the table cell with which this cell is merged?
If yes -> what do you think about storing just the index and not the whole cell?
And a small comment would be nice.

Reply:
Yes. In docframe I just store the first table cell, the merged cells get added to this attribute recursively and removed from the source tree.

The alternative would be to to just store the bool mergedLeft.

I thought it would be easier for convertions to have the first docframe element already having the next table cells.

Reply:
I just saw merged_left in docTYPE - maybe it would be the best to do this also here?
With mergeLeft?

Reply:
like that?
```suggestion
  bool mergeLeft = 6;
```

But for the other case (`ProtoTableCell mergeLeft = 6;`) I'm not sure if it could be missleading, as in doctype the merged_left flag is in the element that should be merged, but in docframe, it contains the elements that are merged with it.

Reply:
I changed it to `bool mergedLeft = 6;` to be in sync with doctype as discussed.